effi-ui-auto-injector.sh

```
@@ -0,0 +1,305 @@
+print_info() { echo -e "${BLUE}ℹ $1${NC}"; }
+
+# Check if file already has Effi-UI
+has_effi_ui() {
+    local file="$1"
+    grep -q "effi_ui_loader.js" "$file" || grep -q "effi_ui_engine.js" "$file"
+}
+
+# Inject Effi-UI into HTML file
+inject_effi_ui() {
+    local file="$1"
+    local filename=$(basename "$file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)